parse_date.re

```diff
@@ -707,24 +707,25 @@ static const timelib_tz_lookup_table* abbr_search(const char *word, timelib_long
 	return NULL;
 }
 
-static timelib_long timelib_lookup_abbr(char **ptr, int *dst, char **tz_abbr, int *found)
+#define TIMELIB_TP_VALUE(tp) tp->gmtoffset - (tp->type * 3600)
+
+static timelib_long timelib_lookup_abbr(char **ptr, int *dst, int is_abbrev, char **tz_abbr, int *found)
```

Comment:
I think this warrants a comment to explain what `is_abbrev` is for, and how it modifies this function.